polkadot/runtime/westend/src/lib.rs

```diff
@@ -1261,6 +1261,9 @@ impl parachains_paras::Config for Runtime {
 	type NextSessionRotation = Babe;
 	type OnNewHead = ();
 	type AssignCoretime = CoretimeAssignmentProvider;
+	type Fungible = Balances;
+	// Per day the cooldown is removed earlier, it should cost 1000.
+	type CooldownRemovalMultiplier = ConstUint<{ 1000 * UNITS / DAYS as u128 }>;
```

Comment:
Is that a suggested configuration for Polkadot? 1000 DOT to unlock an upgrade immediately? Might be worth clarifying in the prdoc.

Reply:
This is a random number I picked. Would not say it is a recommendation for Polkadot.